poetry/core/utils/helpers.py

```diff
@@ -104,3 +104,13 @@ def merge_dicts(d1: dict, d2: dict) -> None:
             merge_dicts(d1[k], d2[k])
         else:
             d1[k] = d2[k]
+
+
+def content_type(path):
```

Comment:
Since this is totally specific to the README/description functionality, let's keep it in the original file. A `@staticmethod` `_readme_content_type` will do the trick.

Also, please add type hints to this method.